code/modules/client/preferences_gear.dm

```diff
@@ -533,7 +537,7 @@ var/global/list/gear_datums = list()
 	path = /obj/item/clothing/under/rank/security/corp
 	cost = 1
 	slot = slot_w_uniform
-	allowed_roles = list("Security Officer","Head of Security","Warden")
+	allowed_roles = list("Security Officer","Head of Security","Warden","Security Cadet","Detective")
```

Comment:
Maybe it'd be good to implement a global define for this, save on duplication and possible mistakes

Would this work?
#define SECURITY_ROLES "Security Officer","Head of Security","Warden","Security Cadet","Detective"

and similar templates for other groups of jobs that are often used